deploy/OneDragon-Installer.spec

```
@@ -9,16 +25,87 @@ a = Analysis(
+        # cryptography 全家（其 DLL 在 cryptography/hazmat/ 子目录下，
+        # 顶层 libssl/libcrypto 是 Python 标准库 _ssl 的依赖，不能按文件名过滤）
+        'cryptography/hazmat/',
+        # PIL 只用基础图像（_imaging），avif 插件 7.5MB 用不到
+        'pil/_avif',
+        # Qt 翻译文件只保留中英文，其余语言 qm 共约 2MB
+        'translations/qt_',
+    )
+    # 只保留 zh_CN 和 en 的 Qt 翻译，其余语言砍掉
+    if 'translations/qt_' in dest and not (
+        dest.endswith('qt_zh_cn.qm') or dest.endswith('qt_en.qm')
+    ):
```

RuboCop (1.88.2)

[fatal] unexpected token tCOLON
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)